Mage/src/main/java/mage/abilities/Modes.java

```diff
@@ -437,8 +456,19 @@ public void addSelectedMode(UUID modeId) {
             throw new IllegalArgumentException("Unknown modeId to select");
         }
 
+        Mode mode = get(modeId);
+
+        if (mode.getPawPrintValue() < 0 && this.getMaxPawPrints() >= 0){
+            throw new RuntimeException("Selected mode has no pawprint value, but this Modes object requires pawprint modes!");
```

Comment:
don't throw RuntimeException, use something like IllegalStateException

Reply:
For info:
- verify check -- can be called from card constructor (by card, ability, effect or other data setup). There are some verify tests that call constructor for all cards and tokens. So such errors will be catches by tests as fast as possible.
- runtime check -- can be called from real game only.

Reply:
To clarify, are you askign for both here? I moved the runtime check pointed to in this thread to a check in addMode() to catch it earlier.